apps/bluetooth/README.md

```diff
@@ -0,0 +1,9 @@
+## Bluetooth
+
+Bluetooth app is responsible for:
+1. Pair Bluetooth devices.
```

Comment:
1. Pairing process for those remote(unpaired) Bluetooth devices. And will pop out attention dialog to interact with user via window.open.
2. Discover Bluetooth devices for user picked in current area. Then, send file(s) to the paired device. If the picked device is unpaired, send file(s) after inline-pairing successfully.